test/error_messages.coffee

```diff
@@ -1126,3 +1126,13 @@ test "cannot import the same member more than once", ->
     import { foo } from 'libB'
              ^^^
   '''
+
+test "imported members cannot be reassigned", ->
+  assertErrorFormat '''
+    import { foo } from 'lib'
+    foo = 'bar'
+  ''', '''
+    [stdin]:2:1: error: 'foo' is read-only
+    foo = 'bar'
+    ^^^
+  '''
```

Comment:
This is good, but you missed this part of my comment (can't blame though you considering how buried this line was among all the other stuff :) ): 

> I think it might work. Just need to clean up and use it for `import a from ''; export a = 2` as well.